src/main/java/core/basesyntax/model/Cat.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.model;
+
+public class Cat {
+    private int age;
+    private String name;
+    private String ownerName;
+
+    @Override
+    public String toString() {
+        return "Cat{"
+                + "age="
+                + age
+                + ", name='"
+                + name
+                + '\''
+                + ", ownerName='"
+                + ownerName
+                + '\''
+                + '}';
```

Comment:
Please, think about a more 'beautiful' format. I think it would be better:
```
        return "Cat{"
                + "age=" + age
                + ", name='" + name + '\''
                + ", ownerName='" + ownerName + '\''
                + '}';
```